timur/test/javascript/utils/query_builder.spec.ts

```diff
@@ -51,86 +28,92 @@ describe('QueryBuilder', () => {
 
   function stamp(model_name: string, attribute_name: string) {
     return {
-      model_name,
-      attribute_name,
-      display_label: `${model_name}.${attribute_name}`
+      model_name, attribute_name, display_label: `${model_name}.${attribute_name}`
     };
   }
 
+  describe('for xcrs1 models', () => {
```

Comment:
I decided to use the actual xcrs1 model metadata here to write the test, so that it maps more cleanly to the original issue.  This pattern is called regression testing and it helps capture common patterns and prevent an existing bug fix from regressing.